Database/QuestieDB.lua

```diff
@@ -935,34 +902,37 @@ function QuestieDB:GetCreatureLevels(quest)
     return creatureLevels
 end
 
-local playerFaction = UnitFactionGroup("player")
+local playerFaction = UnitFactionGroup("player")  -- TODO: FIXME: Api call won't return correct value yet. Only after player entering world event
```

Comment:
Don't forget this TODO 😁 Can't you simply use QuestiePlayer for this?

Reply:
It is a bug in old old old master, I just mentioned it to be a bug 😛